build/android/sign_apk.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 The Brave Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import sys
+import subprocess
+import tempfile
+
+if len(sys.argv) != 8:
+  print("Incorrect number of parameters")
+  print("Usage: " + sys.argv[0] + " <zipalign_path>, <apksigner_path>, <unsigned_apk_path>, <key_path>, <key_passwd>, <prvt_key_passwd>, <key_name>")
+  sys.exit(1)
+
+zipalign_path = sys.argv[1]
+apksigner_path = sys.argv[2]
+unsigned_apk_path = sys.argv[3]
+key_path = sys.argv[4]
+key_passwd = sys.argv[5]
+prvt_key_passwd = sys.argv[6]
+key_name = sys.argv[7]
+
+with tempfile.NamedTemporaryFile() as staging_file:
```

Comment:
I think this stuff should be wrapped in a more typical `Main` method setup to make sure we're getting errors back to gn correctly
```
def Main():
  ...

if __name__ == '__main__':
  sys.exit(Main())
```

Reply:
Done